tokio/src/runtime/builder.rs

```diff
@@ -638,7 +638,7 @@ impl Builder {
     #[cfg(not(loom))]
     pub fn on_thread_park<F>(&mut self, f: F) -> &mut Self
     where
-        F: Fn() + Send + Sync + 'static,
+        F: Fn(usize) + Send + Sync + 'static,
     {
```

Comment:
This is a breaking change. We can't do that.

Reply:
@Darksonn It's a change to an API that is available only when `--cfg tokio_unstable` is used.  I assumed those APIs were subject to modification.  Am I incorrect?

If that's not the case I can certainly add a `on_thread_park2()` method and deprecate the old one (since the new method would have all of the functionality of the old one).  Or how would you suggest evolving the API?

Reply:
You are right that we can change `--cfg tokio_unstable` methods, but the `on_thread_park` method is stable and accessible without `--cfg tokio_unstable`.